### PR TITLE
Add release note 47 and related updates

### PR DESCRIPTION
**Pull request by erkangz**

### CODE REVIEW

File: guides/bulk-upload.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ Navigate to the **Assets** app -> Click the plus (Add Entity) button ->
 Either drag your file into the drop zone, or click the drop zone to select
 your file.
 
+You will be prompted to select (or enter a new) "scope" for the upload.
```
Comment (erkangz) [inline guides/bulk-upload.md:17]: @henrymgarrett FYI

File: docs/schemas/bulk-upload.md [hunk 2]
```diff
@@ -2,9 +2,31 @@
 
 > IMPORTANT: Bulk upload triggers a data synchronization process that
 > automatically **updates** or **deletes** entities/relationships as needed.
+> Previously existing entities/relationships within the same `_scope` that no
```
Comment (ndowmon) [inline docs/schemas/bulk-upload.md:5]: Should this be `_scope` or `scope`?

File: guides/bulk-upload.md (hunk 1 above)
Comment (henrymgarrett) [reply guides/bulk-upload.md:17]: Perfect, thanks Erkang

File: docs/schemas/bulk-upload.md (hunk 2 above)
Comment (erkangz) [reply docs/schemas/bulk-upload.md]: good catch. fixed.

**Pull request merged**